src/core/cgroup.h

```diff
@@ -32,6 +32,23 @@ typedef enum CGroupDevicePolicy {
         _CGROUP_DEVICE_POLICY_INVALID = -1
 } CGroupDevicePolicy;
 
+typedef enum CGroupControlEvent {
+        CGROUP_EMPTY,
+        CGROUP_FROZEN,
+        CGROUP_THAW,
```

Comment:
Hmm, shouldn't this be `CGROUP_THAWED`, since it indicates an end state?

Reply:
Agreed.

Reply:
Resolved in latest version